icu4c/source/common/wintz.cpp

```diff
@@ -34,221 +32,33 @@
 #   define NOMCX
 #include <windows.h>
 
-#define MAX_LENGTH_ID 40
-
-/* The layout of the Tzi value in the registry */
-typedef struct
-{
-    int32_t bias;
-    int32_t standardBias;
-    int32_t daylightBias;
-    SYSTEMTIME standardDate;
-    SYSTEMTIME daylightDate;
-} TZI;
+#define MAX_LENGTH_ID 128
```

Comment:
Maybe add a comment for this define to indicate where the value comes from?
Nit: Also I wonder if `MAX_TIMEZONE_ID_LENGTH` might be slightly more clear? (Up to you)

Reply:
Comments added. Thanks!